packages/create-app/src/createApp.ts

```diff
@@ -107,7 +107,7 @@ export default async (cmd: Command): Promise<void> => {
   ];
   const answers: Answers = await inquirer.prompt(questions);
 
-  const templateDir = paths.resolveOwn('templates/default-app');
+  const templateDir = paths.resolveOwnRoot('packages/cli/templates/default-app');
```

Comment:
So this won't be available when running via `npx`, as it assumes this monorepo layout we'll be executed as a package in `node_modules`. I'd say just move over the `default-app` template to this package instead. It does create a problem with the dependency on the template in `app:diff`, but tbh I think we can remove that for now as it isn't very useful in its current form anyway.